test/e2e/config/fixture.yaml

```diff
@@ -23,7 +23,9 @@ variables:
   UPGRADE_DEPLOY_IRONIC: "false"
   UPGRADE_DEPLOY_BMO: "true"
   UPGRADE_DEPLOY_CERT_MANAGER: "true"
-  UPGRADE_BMO_KUSTOMIZATION_FROM: "../../config/overlays/fixture-release-0.5"
+  UPGRADE_BMO_KUSTOMIZATION_FROM: "../../config/overlays/fixture-release-0.3 ../../config/overlays/fixture-release-0.4 ../../config/overlays/fixture-release-0.5"
```

Comment:
I think it is time to turn this into proper structured config. It is becoming too messy. Could you take a look at how this looks for CAPI and try to do something similar?

Reply:
Yep. Good idea! I'll have a look